### PR TITLE
Css basics

### PR DESCRIPTION
**Pull request by VanessaSchwab1906**
styled all the buttons

### CODE REVIEW

**Review by WinterIsCoding23 — COMMENTED** [review]
Sorry, can't find css-stylesheet? Not committed yet perhaps? Or am I looking in the wrong place? :-D

**Comment by VanessaSchwab1906** [comment]
> Sorry, can't find css-stylesheet? Not committed yet perhaps? Or am I looking in the wrong place? :-D

You're right, the css-file is missing... I'll try to push it/fix it/whatever (github is fun! :)))) )

File: css-basics_button-styles/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+.button2 {
+  /*Style me! Change my color, or my background-color. Or both! You can test out different color modes here - like HEX, RGB, ... */
+  color: #ad7be9;
+  background-color: #bef0cb;
+  border-color: #ad7be9;
+}
+
+.button3 {
+  /*Style me! You can test out changing the font, or font size, ...*/
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
```
Comment (WinterIsCoding23) [inline css-basics_button-styles/css/styles.css:51]: Bonus-point for fancy font-names!

**Review by WinterIsCoding23 — APPROVED** [review]
Great work!

**Pull request merged**